spec/support/active_record.rb

```diff
@@ -6,13 +6,25 @@
 
 FileUtils.mkdir_p('log')
 
+puts Travis.config.database.to_h
+puts Travis.config.logs_database.to_h
+exit 1
```

Comment:
I would add puts "*******" or something you can search for instead of exit - also puts I believe only shows up in the heroku logs not in papertrail, however you should only see this output at startup.